File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/02_Conditional_Statements_Exercise/05_Godzilla_vs_Kong.js
function godzillaVsKong(input) {
    let budget = Number(input[0]);
    let statists = Number(input[1]);
    let clothesPrice = Number(input[2]);

    let decor = budget * 0.10;
    let clothesAll = statists * clothesPrice;

    // console.log(clothesAll + " predi");
    if (statists > 150) {
        clothesAll *= 0.90;
        // console.log(clothesAll + " sled");
    }

    let expenses = decor + clothesAll;

    if (expenses <= budget) {  // budget < expenses
        console.log('Action!');
        console.log(`Wingard starts filming with ${(budget - expenses).toFixed(2)} leva left.`);
    } else {
        console.log('Not enough money!');  // "" '' ``
        console.log(`Wingard needs ${(expenses - budget).toFixed(2)} leva more.`);
    }
}


// godzillaVsKong(["15437.62", "186", "57.99"]);

File: Python_Courses/Python_Basics/Python_PB_2025_01/02_Conditional_Statements_Exercise_OFF/05_Godzilla_vs_Kong.py
budget = float(input())
walk_on_count = int(input())
clothes_price = float(input())

# •	Декорът за филма е на стойност 10% от бюджета.
decor = budget * 0.10

total_clothes_price = walk_on_count * clothes_price

# •	При повече от 150 статиста,  има отстъпка за облеклото на стойност 10%.
if walk_on_count > 150:
    total_clothes_price *= 0.90

total_cost = decor + total_clothes_price

if budget >= total_cost:
    money_left = budget - total_cost
    print('Action!')
    print(f'Wingard starts filming with {money_left :.2f} leva left.')
else:
    money_needed = total_cost - budget
    print('Not enough money!')
    print(f'Wingard needs {money_needed :.2f} leva more.')
